webapp/app.py

```diff
@@ -4,7 +4,7 @@
 The web frontend for the snap store.
 """
 
-import flask
+from canonicalwebteam.flask_base.app import FlaskBase
 from werkzeug.contrib.fixers import ProxyFix
 from werkzeug.debug import DebuggedApplication
```

Comment:
This too: https://github.com/canonical-web-and-design/canonicalwebteam.flask-base/blob/master/canonicalwebteam/flask_base/app.py#L56